Tests/FunctionalTests/ModuleAliasingTests.swift

```diff
@@ -29,9 +32,14 @@ class ModuleAliasingTests: XCTestCase {
             XCTAssertFileExists(prefix.appending(components: "AppPkg", ".build", UserToolchain.default.triple.platformBuildPathComponent(), "release", "GameUtils.swiftmodule"))
             XCTAssertFileExists(prefix.appending(components: "AppPkg", ".build", UserToolchain.default.triple.platformBuildPathComponent(), "debug", "Utils.swiftmodule"))
             XCTAssertFileExists(prefix.appending(components: "AppPkg", ".build", UserToolchain.default.triple.platformBuildPathComponent(), "release", "Utils.swiftmodule"))
+            
             let result = try SwiftPMProduct.SwiftBuild.executeProcess([], packagePath: app)
-            XCTAssertEqual(result.exitStatus, .terminated(code: 0))
+            let output = try result.utf8Output() + result.utf8stderrOutput()
+           
+            // FIXME: rdar://88722540
+            // The process from above crashes in a certain env, so print
+            // the output for further investigation
+            print(output)
```

Comment:
Would it make sense to still mark as skipped if it fails here (after emitting the output)?  Otherwise it looks as if it passed even if the inferior crashed.

I think all that's wanted here, temporarily, while investigating this was to:
- emit the output if it fails
- treat it as skipped and not failure

The current code would mark as success whether or not the inferior crashes, which seems unfortunate.

Reply:
+1 on this idea

Reply:
makes sense; updated